BlazarService/src/main/java/com/hubspot/blazar/integration/slack/SlackClient.java

```diff
@@ -69,7 +69,7 @@ public void sendMessage(final SlackMessage message) {
       }
     }
     if (message.getIconEmoji().isPresent()) {
-      requestBuilder.setFormParam("icon_emoji").to(message.getIconEmoji().get());
+      requestBuilder.setFormParam("icon_emoji").to(message.getIconEmoji().or(":fire:"));
```

Comment:
I run with slack in `compact` mode. Didn't realize some of the messages we send don't have icons.
Set it to <img width="51" alt="screen shot 2016-04-19 at 11 19 33 am" src="https://cloud.githubusercontent.com/assets/3247956/14644607/9be90792-0620-11e6-8ddf-a1655b67acde.png"> instead of  <img width="54" alt="screen shot 2016-04-19 at 11 19 26 am" src="https://cloud.githubusercontent.com/assets/3247956/14644608/9be9aec2-0620-11e6-962e-244e73ccbe25.png">.